.github/workflows/tests-integration.yml

```diff
@@ -24,6 +24,11 @@ on:
       - "Tests"
     types:
       - completed
+  push:
+    branches:
```

Comment:
question: Is it possible to run integration tests only if the unit tests pass successfully?

Reply:
At this point we decided only to run integration tests on push (after the pr is merged) This is a little bit strange but in reality, this was what was happening with the bug we found yesterday.
This solution is a stop-gap solution and I hope we can start the work to re-add the integration tests into the PR workflow at the beginning of next week.